### PR TITLE
ft: ZENKO-717: replicationBackends constant moved to Arsenal

### PR DESCRIPTION
**Pull request by jeremyds**
With the management code moved from cloud server to its own repository,
this constant should be shared in Arsenal constants.

# Pull request template

### Motivation and context

Why is this change required? What problem does it solve?

### Related issues

Please use the following link syntaxes #600 to reference issues in the
current repository

### Add tests to cover the changes

New tests added or existing tests modified to cover all changes

### Sign your work

In order to contribute to the project, you must sign your work
https://github.com/scality/Guidelines/blob/master/CONTRIBUTING.md#sign-your-work

Thank you again for contributing! We will try to test and integrate the change
as soon as we can.

### CODE REVIEW

File: lib/management/configuration.js [hunk 1]
```diff
@@ -13,6 +12,7 @@ const managementDatabaseName = 'PENSIEVE';
 const replicatorEndpoint = 'zenko-cloudserver-replicator';
 const { decryptSecret } = arsenal.pensieve.credentialUtils;
 const { reshapeExceptionError } = arsenal.errorUtils;
+const replicationBackends = arsenal.constants.replicationBackends;
```
Comment (nicolas2bert) [inline lib/management/configuration.js]:
```suggestion
const { replicationBackends } = arsenal.constants;
```

File: lib/api/apiUtils/object/getReplicationInfo.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 const s3config = require('../../../Config').config;
-const constants = require('../../../../constants');
 const { isBackbeatUser } = require('../authorization/aclChecks');
+const replicationBackends = require('arsenal').constants.replicationBackends;
```
Comment (nicolas2bert) [inline lib/api/apiUtils/object/getReplicationInfo.js]:
```suggestion
const { replicationBackends }  = require('arsenal').constants;
```

**Review by nicolas2bert — APPROVED** [review]

**Review by philipyoo — APPROVED** [review]

File: lib/management/configuration.js (hunk 1 above)
Comment (jeremyds) [reply lib/management/configuration.js]: done

File: lib/api/apiUtils/object/getReplicationInfo.js (hunk 2 above)
Comment (jeremyds) [reply lib/api/apiUtils/object/getReplicationInfo.js]: done

**Review by rahulreddy — APPROVED** [review]

**Comment by rahulreddy** [comment]
@bert-e approve

**Comment by jeremyds** [comment]
@bert-e status

**Comment by jeremyds** [comment]
@bert-e approve

**Pull request merged**